### ISSUE TITLE
vmagent add 80 port to any snmp_exporter targets by default

### ISSUE THREAD

**Issue by v98765** [VictoriaMetrics/VictoriaMetrics#331]
snmp_exporter log:
`caller=collector.go:216 module=cisco_l2 target=cisco1:80 msg="Error scraping target" err="error getting target cisco1: context canceled"`

**Comment by valyala** [comment]
@v98765 , could you provide sample config used in Prometheus, which works unexpectedly with vmagent?

**Comment by valyala** [comment]
It looks like I figured out the issue and fixed it in 7a33da8fead6667cbd056e6cdb7f01fa6ba6d2e9 . `vmagent` was adding missing TCP port to `__address__` label, which could result in incorrect scrape url after relabeling defined at https://github.com/prometheus/snmp_exporter/blob/master/README.md#prometheus-configuration . Now it should generate correct scrape url in this case.

@v98765 , could you build `vmagent` from this commit according to [these docs](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmagent/README.md#how-to-build-from-sources) and verify whether the issue is fixed?

**Comment by v98765** [comment]
issue is fixed
vmagent-20200226-022202-v1.34.0-7-g7a33da8
```
curl http://localhost:8429/targets
job="cisco_l2" (2/2 up)
        state=up, endpoint=http://127.0.0.1:9116/snmp?module=cisco_l2&target=cisco1, labels={instance="cisco1", job="cisco_l2", obj="obj1"}, last_scrape=17.196s ago, scrape_duration=0.192s, error=""
```

**Comment by valyala** [comment]
The bugfix is available starting from [v1.34.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases)
